spec/requests/users/registrations/register_spec.rb

```diff
@@ -90,5 +90,24 @@
         } }
       }.to raise_error(ActionController::BadRequest)
     end
+
+    context "when school has old unconfirmed registrations" do
+      let(:user) { create(:user, confirmed_at: nil) }
+      let!(:coordinator) { create(:induction_coordinator_profile, user: user, schools: [school]) }
+      let(:created_user) { User.find_by_email(email) }
```

Comment:
I think for a single test we can put
```
      created_user = User.find_by_email(email)
```
in the test.

On the other hand, that line is repeated a few times throughout this test file - so maybe we want `created_user` to be defined and used one context higher?